mappings/net/minecraft/advancement/AdvancementCriterion.mapping

```diff
@@ -1,6 +1,16 @@
 CLASS net/minecraft/unmapped/C_rzypsigz net/minecraft/advancement/AdvancementCriterion
-	FIELD f_bjqitkog conditions Lnet/minecraft/unmapped/C_mysyaxfs;
+	FIELD f_bjqitkog trigger Lnet/minecraft/unmapped/C_mysyaxfs;
+	FIELD f_yiamcoxx conditions Lnet/minecraft/unmapped/C_zkhebbek;
 	METHOD m_bdrggclo criteriaFromJson (Lcom/google/gson/JsonObject;Lnet/minecraft/unmapped/C_gzewtdot;)Ljava/util/Map;
-		ARG 0 obj
+		ARG 0 jsonObject
```

Comment:
Personally i prefer `json` over `obj` and `jsonObject`. What do others think?

Reply:
I chose to unify them under `jsonObject` due to it being the default name when obfuscated. If we prefer `json` or `obj` (I'd rather we choose `json`), perhaps that should be the default name

Reply:
I slightly prefer `json` to `jsonObject` here, just because it matches the method name (`obj` is bad, imo).
I think it could make sense to have a convention where `json` is used for `*FromJson` method params, otherwise use `jsonObject` (or something more descriptive if possible).